src/web_interface/filter.py

```diff
@@ -400,3 +401,11 @@ def elapsed_time(start_time: float) -> int:
 
 def format_duration(duration: float) -> str:
     return str(timedelta(seconds=duration))
+
+
+def render_query_title(query_title: str):
```

Comment:
Is `query_title` really of type `str` when there are checks whether it is `None` or `dict`?

Reply:
Good point. It's either None, a string or a dict

Reply:
Though no idea if the None case actually exists, but it was handled before.